Gulpfile.js

```diff
@@ -1175,3 +1175,13 @@ export function downloadSidecar() {
 
   return result;
 }
+
+prepareHusky.description = "Initialize Husky Git hooks for pre-commit linting.";
+export function prepareHusky(done) {
+  const result = spawnSync("npx", ["husky"], {
+    stdio: "inherit",
```

Comment:
this apparently passes the child process's stdin, stdout, and stderr directly to the parent process to allow you to see the Husky command's output in the terminal